scripts/release/version.sh

```diff
@@ -2,7 +2,9 @@
 
 set -o errexit
 
-scripts/release/update-changelog-release-date.js
+npx changeset version
```

Comment:
We assume that this runs through `npm run version`, so the PATH is already correctly set.

```suggestion
changeset version
```

Reply:
Thanks!

Reply:
Thanks!